app/Support/IcsGenerator.php

```
@@ -280,30 +280,102 @@ private function formatEvent(array $event): string
 
+    /**
+     * Application timezone for interpreting DB timestamps
+     *
+     * Resolves ConfigStore 'app.timezone' like DateHelper does, falling back to
+     * the constructor timezone and finally to Europe/Rome if invalid.
+     *
+     * @return \DateTimeZone Resolved timezone object
+     */
+    private function appTimezone(): \DateTimeZone
+    {
+        $tz = (string) ConfigStore::get('app.timezone', $this->timezone);
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Support\ConfigStore' in method 'appTimezone'. (undefined)

(StaticAccess)